app/src/main/java/org/apache/fineract/ui/online/customers/createcustomer/customeractivity/CreateCustomerActivity.java

```diff
@@ -204,6 +205,20 @@ public void showError(String message) {
         Toaster.show(findViewById(android.R.id.content), message);
     }
 
+    @Override
+    public void onBackPressed() {
+        new MaterialDialog.Builder()
```

Comment:
It is a good feature to have in the App but you have to handle other scenarios too like we shouldn't show this dialog if the user has not entered any data or didn't edit any existing data which we are using the same fragments for editing and updating too.

Reply:
Are you sure, we want to check that? Because we are only notifying the user that only the 'unsaved' changes would be lost.

Reply:
Yeah I think it is necessary because showing a dialog to the user saying that he will lose his unsaved data even if he didn't enter any data or changed anything, kind of puts the user in doubt and it's unnecessary, and also it is not good from UX point of view. You can implement this by setting a `touch listener` to EditTexts and handle it using a `boolean`.